api_examples/graphql/queries/gcp_project_import.graphql

```diff
@@ -0,0 +1,116 @@
+# Description
+# -----------
+#
+# The mutation query import GCP project to a Turbot Folder. The mutation query needs to be run in 2 steps.
+# STEP 1 is to create a GCP project inside the Turbot Folder.
+# STEP 2 is to use the Turbot ID that is returned from STEP 1 as resource ID for setting client email and privet key.
+
+# Prerequisites
+# -------------
+#
+# Create a service account in IAM for the project that needs to be imported. 
+# After the service account has been created, assign the following roles: owner,viewer,Pub/Sub Admin and Logging Admin.
+# Generate an access key for the service account. Once you select the key format as 'JASON', a document will 
+# automatically get downloaded, which contains all the details including the private key.
+# In order for Turbot to be able to manage services in the project, admins must enable the Cloud Resource Manager 
+# and Service Management APIs.
+
+# Substitute
+# ----------
+#
+# Substitute real values for STEP 1.
+#
+# <parent>:
+#   Any valid Turbot folder ID can be substituted.
+#   To get Turbot folder ID, please see the perticular Turbot env where the account is suppose to be imported.
```

Comment:
perticular -> particular

Reply:
Turbot env where the account is suppose to be imported.

Turbot environment where the account is imported.